tests/unit_tests/toporder/test_score.py

```diff
@@ -0,0 +1,261 @@
+import networkx as nx
+import numpy as np
+import pandas as pd
+import pytest
+
+from dodiscover import make_context
+from dodiscover.metrics import structure_hamming_dist, toporder_divergence_arr
+from dodiscover.toporder.score import SCORE
+from dodiscover.toporder.utils import full_DAG, orders_consistency
+
+
+# -------------------- Fixtures -------------------- #
+@pytest.fixture
+def dummy_sample():
+    """
+    Ground truth:  [[0, 0, 0, 0],
+                   [1, 0, 0, 0],
+                   [1, 0, 0, 1],
+                   [1, 0, 0, 0]]
+    """
+    X = np.array(
+        [
+            [-0.839573121, -0.277701505, -0.430597219, -0.060861940],
```

Comment:
These data are synthetically generated from code that I usually use for my research. I think that I would avoid adding that here since it requires some additional dependencies - unless you think it's fine, I don't have strong opinions on this. I can try generate similar data with the AdditiveNoiseModel class from the dowhy.gcm.fcms module, what do you think @adam2392?

Reply:
Re adding your code: Yes prolly best to avoid adding here, unless they are simple? We could have them in the test file as private data-generating functions.

Re try generating similar data: If it's easy to do, that would be great, just for making the unit-tests a bit more readable/simplified. You can also demonstrate this in the example then, which would be informative.

Reply:
In the end I opted for writing some data generator function `dummy_sample` in `dodiscover.toporder.utils.py` that takes as input a random seed.